.circleci/config.yml

```diff
@@ -31,20 +43,29 @@ jobs:
     steps:
     - checkout
 
-    - run: |
-        wget -q $(curl -sS https://api.github.com/repos/giantswarm/architect/releases/latest | grep browser_download_url | head -n 1 | cut -d '"' -f 4)
-        chmod +x ./architect
-        ./architect version
-
-    - run: ./architect build
+    - run:
+        name: Install architect (except when in cronjob execution)
+        command: |
+            if [ "${CIRCLE_USERNAME}" != "" ]; then
+                wget -q $(curl -sS -H "Authorization: token $RELEASE_TOKEN" https://api.github.com/repos/giantswarm/architect/releases/latest | grep browser_download_url | head -n 1 | cut -d '"' -f 4)
+                chmod +x ./architect
+                ./architect version
+            fi
+
+    - run:
+        name: architect build (except when in cronjob execution)
+        command: test -x ./architect && ./architect build
+
+    - deploy:
+        name: architect deploy (master only, except when in cronjob execution)
+        command: |
+            if [ "${CIRCLE_BRANCH}" == "master" ]; then
```

Comment:
Do we need to test for cronjob execution here?

Reply:
If architect is in place (test -x ./architect) we are not in a cronjob

Reply:
I get this now